src/elisp/treemacs-branch-creation.el

```diff
@@ -99,6 +107,8 @@ DEFAULT: Face"
           (-pcase treemacs-sorting
             [`alphabetic-asc  #'treemacs--sort-alphabetic-asc]
             [`alphabetic-desc #'treemacs--sort-alphabetic-desc]
+            [`alphabetic-case-insensitive-asc  #'treemacs--sort-alphabetic-case-insensitive-asc]
+            [`alphabetic-case-insensitive-desc #'treemacs--sort-alphabetic-case-insensitive-desc]
             [`size-asc        #'treemacs--sort-size-asc]
```

Comment:
You should also delete the extra whitespace in the other match arms. I try to keep things nicely aligned, but not if the difference is as large as with the new values.